samples/net/updatehub/README.rst

```diff
@@ -77,19 +75,19 @@ following Docker command:
 
     docker run -d -p 8080:8080 -p 5683:5683/udp --rm  updatehub/updatehub-ce-server:latest
 
-Using this server you can set the option :option: ``CONFIG_UPDATEHUB_SERVER``
-on the ``prj.conf`` file with your local ip address and the option :option:
-``CONFIG_UPDATEHUB_CE_SERVER`` with true.
+Using this server you can set the option :option:``CONFIG_UPDATEHUB_SERVER``
+on the ``prj.conf`` file with your local ip address and the option
```

Comment:
Please instruct user to create overlay-prj.conf file here, and place all the bits and pieces related to his setup there.